jenkins-builds/MCH-RH7.4-17.02

```diff
@@ -0,0 +1,47 @@
+ Anaconda3-4.4.0.eb                                  --installpath=/apps/escha/UES/jenkins/RH7.4/generic/easybuild
+ Autotools-201602.eb                                 --installpath=/apps/escha/UES/jenkins/RH7.4/generic/easybuild
+ Boost-1.63.0-gmvolf-17.02-Python-2.7.13.eb
+ c2sm-rcm-1.00.00-kesch.eb                           --set-default-module --installpath=/apps/escha/UES/jenkins/RH7.4/generic/easybuild
+ CDO-1.8.2-gmvolf-17.02.eb
+ CMake-3.13.0.eb                                     --installpath=/apps/escha/UES/jenkins/RH7.4/generic/easybuild
+ ddt-18.2.2-Redhat-7.0.eb                            --set-default-module --installpath=/apps/escha/UES/jenkins/RH7.4/generic/easybuild
+ GDAL-2.2.1-gmvolf-17.02.eb
+ GSL-1.16-gmvolf-17.02.eb
+ geeqie-1.3.eb                                       --installpath=/apps/escha/UES/jenkins/RH7.4/generic/easybuild
+ git-2.19.2.eb                                       --installpath=/apps/escha/UES/jenkins/RH7.4/generic/easybuild
+ HDF-4.2.13-gmvolf-17.02.eb
+ IDL-8.5.1.eb                                        --installpath=/apps/escha/UES/jenkins/RH7.4/generic/easybuild
+ ImageMagick-7.0.8-14.eb                             --installpath=/apps/escha/UES/jenkins/RH7.4/generic/easybuild
+ NCL-6.5.0.eb                                        --installpath=/apps/escha/UES/jenkins/RH7.4/generic/easybuild
+ NCO-4.6.2-gmvolf-17.02.eb
+ nmon-16g.eb                                         --installpath=/apps/escha/UES/jenkins/RH7.4/generic/easybuild
+ PrgEnv-gnu-kesch-gmvolf-17.02.eb                    --module-naming-scheme=EasyBuildMNS
+ gmvapich2-17.02_cuda_8.0_gdr.eb
+ matlab-r2016a.eb                                    --installpath=/apps/escha/UES/jenkins/RH7.4/generic/easybuild
+ Python-2.7.13-gmvolf-17.02.eb
+ Python-3.6.2-gmvolf-17.02.eb
+ R-3.3.2-gmvolf-17.02-libX11-1.6.3.eb
+ R-3.5.2-gmvolf-17.02-libX11-1.6.3.eb
+ reframe-2.14.eb                                     --set-default-module
+ Ruby-2.5.3-gmvolf-17.02.eb
+ ncview-2.1.7-gmvolf-17.02.eb
+ netCDF-C++-4.3.0-gmvolf-17.02.eb
+ netCDF-Fortran-4.4.4-gmvolf-17.02.eb
+ netcdf4-python-1.3.0-gmvolf-17.02-Python-2.7.13.eb
+ netcdf4-python-1.3.0-gmvolf-17.02-Python-3.6.2.eb
+ netCDF-4.6.1-PGI-18.10-GCC-5.4.0-2.26.eb            --installpath=/apps/escha/UES/jenkins/RH7.4/pgi_PE18.10/easybuild 
+ netCDF-C++-4.3.0-PGI-18.10-GCC-5.4.0-2.26.eb        --installpath=/apps/escha/UES/jenkins/RH7.4/pgi_PE18.10/easybuild
+ netCDF-Fortran-4.4.4-PGI-18.10-GCC-5.4.0-2.26.eb    --installpath=/apps/escha/UES/jenkins/RH7.4/pgi_PE18.10/easybuild
+ PrgEnv-pgi-kesch-18.10.eb                           --installpath=/apps/escha/UES/jenkins/RH7.4/pgi_PE18.10/easybuild --module-naming-scheme=EasyBuildMNS
```

Comment:
Is it possible to sort the file? It is a bit though to identify the installed packages.

Reply:
@victorusu I kept the original ordering, we can certainly re-arrange the items though.